translations/wesnoth-A_New_Order/es.po

```diff
@@ -2382,25 +2486,25 @@ msgstr "¿Y por qué.. debo hacerlo?"
 #: A_New_Order/scenarios/18_Start_of_the_Quest.cfg:232
 #: A_New_Order/scenarios/18_Start_of_the_Quest.cfg:382
 #: A_New_Order/scenarios/18_Start_of_the_Quest.cfg:493
+#, fuzzy
```

Comment:
Why is this (and many similar short and good-looking translations) marked as fuzzy?
```suggestion
```

Reply:
They are some of the previous translations. I have to go through all of them, because about 75% of them look like machine translations, and about 10% are old translations that require updating because the original strings changed at some later point.
In any case, the previous translation is only 17% complete, the most incomplete of all that existed, so it is not very relevant either.